src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseFileServiceTests.cs

```diff
@@ -1740,38 +1778,228 @@ await using (var contentDbContext = InMemoryApplicationDbContext())
         }
 
         [Fact]
-        public async Task UpdateName()
+        public async Task Update()
         {
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+            };
+
             var releaseFile = new ReleaseFile
             {
-                Release = new Release(),
+                Release = release,
                 Name = "Test PDF File",
-                File = new File()
+                File = new File
+                {
+                    RootPath = release.Id,
+                    Filename = "test.pdf",
+                    Type = Ancillary,
+                    Created = new DateTime(),
+                    CreatedBy = new User
+                    {
+                        Email = "test@test.com"
+                    }
+                }
             };
 
-            var contentDbContextId = Guid.NewGuid().ToString();
-            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            var contextId = Guid.NewGuid().ToString();
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
             {
                 await contentDbContext.AddAsync(releaseFile);
                 await contentDbContext.SaveChangesAsync();
             }
 
-            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
             {
-                var service = SetupReleaseFileService(contentDbContext);
+                var service = SetupReleaseFileService(contentDbContext: contentDbContext);
 
-                var result = await service.UpdateName(releaseFile.ReleaseId, releaseFile.FileId, "New file title");
+                var result = await service.Update(
+                    releaseFile.ReleaseId,
+                    releaseFile.FileId,
+                    new ReleaseFileUpdateViewModel
+                    {
+                        Title = "New file title",
+                        Summary = "New file summary"
+                    }
+                );
 
                 Assert.True(result.IsRight);
```

Comment:
Doesn't really gain you anything here, but just so you're aware, you can also do:

`var unit = result.AssertRight()`

To both assert and extract the contents of the right in one go.

Reply:
Done!